resolwe/flow/models/entity.py

```diff
@@ -29,6 +29,9 @@ class Meta(BaseCollection.Meta):
     #: indicate whether `descriptor` is completed (set by user)
     descriptor_completed = models.BooleanField(default=False)
 
+    #: tags for categorizing objects
+    tags = ArrayField(models.CharField(max_length=255), default=list)
```

Comment:
Can you pass a `set` for the default callable or does Django have something like a SetField? Tag values are unique.